hvplot/utilities.py

```diff
@@ -31,9 +31,8 @@ def show(obj, title=None, port=0, **kwargs):
 
 class hvplot_extension(_hv.extension):
 
-    compatibility = param.Selector(
-        default=None, objects=['bokeh', 'matplotlib', 'plotly'],
-        allow_None=True, doc="""
+    compatibility = param.ObjectSelector(
+        allow_None=True, objects=['bokeh', 'matplotlib', 'plotly'], doc="""
             Backend used to parse styling options.""")
```

Comment:
Maybe from a user perspective it should be something like 'Plotting library API used to process extra keyword arguments' (or similar)?